signalfx-tracing/signalfx-dotnet-tracing/cloudfoundry-buildpack/manifest.yml

```diff
@@ -0,0 +1,7 @@
+applications:
+- name: "dotnet-cf-demo"
+  buildpacks:
+  - "signalfx_dotnet_tracing_buildpack"
+  - "dotnet_core_buildpack"
+  env:
+    SIGNALFX_TRACE_LOG_PATH: "/home/vcap/sfx-dotnet-agent.log"
```

Comment:
Do we have the Smart Agent here?

Reply:
Not yet, I was planning to test both Java and .Net examples E2E with Smart Agent later on.